pkg/cri/resource-manager/cache/container.go

```diff
@@ -790,3 +793,20 @@ func (c *container) HasPending(controller string) bool {
 	_, pending := c.pending[controller]
 	return pending
 }
+
+func (c *container) GetTag(key string) (string, bool) {
+	value, ok := c.Tags[key]
+	return value, ok
+}
+
+func (c *container) SetTag(key string, value string) (string, bool) {
+	prev, ok := c.Tags[key]
+	c.Tags[key] = value
+	return prev, ok
+}
+
+func (c *container) DeleteTag(key string) (string, bool) {
+	prev, ok := c.Tags[key]
```

Comment:
nit: `prev` is a bit strange name in this context :smile:

Reply:
True, I'll change it to `value`...